test/same-shape.test.js

```diff
@@ -0,0 +1,64 @@
+'use strict'
+
+const { test } = require('tap')
+const fastify = require('..')
+
+test('same shape on Request', async (t) => {
+  t.plan(1)
+
+  const app = fastify()
+
+  let request
```

Comment:
My head is too foggy right now to understand everything from the discussion, but I wondered if this closure var affects the results?

Reply:
no it doesn't in this specific case.